vars/functionalTestPostV2.groovy

```diff
@@ -98,8 +98,15 @@ void call(Map config = [:]) {
            script: 'ci/functional/launchable_analysis "' + fileName + '"')
     }
 
+    String script = 'if pip3 install'
+    if (env.DAOS_HTTPS_PROXY) {
+        script += ' --proxy "' + env.DAOS_HTTPS_PROXY + '"'
+    }
+    script += ' --user --upgrade launchable~=1.0'
```

Comment:
The pip program searches a number of locations for a config file name pip.conf or something similar.
When we setup a docker container or a system to use pip, we should be updating that file instead of adding --proxy to the pip command.